projects/ngx-mat-popover/package.json

```diff
@@ -1,14 +1,12 @@
 {
   "name": "ngx-mat-popover",
   "description": "angular material simple popover",
-  "version": "0.0.1",
-  "author": {
-    "name": "Sam Lin",
-    "email": "maxisam@gmail.com"
-  },
+  "version": "0.0.2",
   "homepage": "https://github.com/maxisam/ngx-mat-popover",
   "license": "MIT",
-  "publishConfig": {},
+  "publishConfig": {
```

Comment:
should we change this?

Reply:
No sorry it's a mistake when i was trying the lib in another project. Reverting to original